### ISSUE TITLE
Calculation of location of eslintrc.json file for libraries generated by schematics is not correct

### ISSUE THREAD

**Issue by trdyer** [angular-eslint/angular-eslint#879]
I am using the @angular-eslint/schematics:library schematic to generate a 'scoped' library. ie `my-org/my-library`

Doing this will call [createESLintConfigForProject](https://github.com/angular-eslint/angular-eslint/blob/2000c0c63f7b2c4d67c387cce4be8d039885da40/packages/schematics/src/utils.ts#L343) In my case there isn't a root application but I do have a .eslintrc.json file in place in the root of the project structure.

Which will call [createProjectESLintConfig](https://github.com/angular-eslint/angular-eslint/blob/2000c0c63f7b2c4d67c387cce4be8d039885da40/packages/schematics/src/utils.ts#L302) which will call [offsetFromRoot](https://github.com/angular-eslint/angular-eslint/blob/2000c0c63f7b2c4d67c387cce4be8d039885da40/packages/schematics/src/utils.ts#L127).

I believe that `offsetFromRoot` is provided an argument of the root of the angular project file tree, which should always be `/`. This means the location of the file that the project rc file extends will always be `../../.eslintrc.json`. 

If you are using scoped libraries you would likely have a path like `projects/my-org/my-library` and want to navigate up three directories.

In addition if the root path wasn't `/` then I believe the calculation would also be off. For instance a project location of `/foo/bar/` would require `../../../foo/bar/.eslintrc.json` in my example of using a scoped library.

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/utils`            | `13.0.1` |
| `ESLint`                           | `7.32.0` |
| `node`                             | `16.11.1` |

```sh
# Please run `npx ng version` in your project and paste the full output here:
     _                      _                 ____ _     ___
    / \   _ __   __ _ _   _| | __ _ _ __     / ___| |   |_ _|
   / △ \ | '_ \ / _` | | | | |/ _` | '__|   | |   | |    | |
  / ___ \| | | | (_| | |_| | | (_| | |      | |___| |___ | |
 /_/   \_\_| |_|\__, |\__,_|_|\__,_|_|       \____|_____|___|
                |___/
    

Angular CLI: 12.2.15
Node: 16.11.1
Package Manager: npm 8.0.0
OS: darwin x64

Angular: 12.2.15
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1002.1
@angular-devkit/build-angular   12.2.15
@angular-devkit/core            10.2.1
@angular-devkit/schematics      13.0.1
@angular/cli                    12.2.15
@schematics/angular             12.2.15
ng-packagr                      12.2.6
rxjs                            6.6.7
typescript                      4.3.5
```

- [x] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by JamesHenry** [comment]
Thanks @trdyer, however I was not able to reproduce any issues at multiple levels of nesting with the latest version.

I will close this for now but please feel free to reopen if you can provide steps to reproduce
